ruby/gilded_rose.rb

```diff
@@ -65,4 +24,128 @@ def initialize(name, sell_in, quality)
   def to_s()
     "#{@name}, #{@sell_in}, #{@quality}"
   end
+end
+
+class ItemUpdaterFactory
+  def self.create_updater_for(item)

```

Comment:
A nit-pick: I think the verb "build" is more often used in the context of factories, instead of "create".